### PR TITLE
[FIX] 'product_by_supplier' add pot file; [ADD] 'product_by_supplier' fr translation;

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: 

Thanks.

**Comment by eLBati** [comment]
:+1:

File: product_by_supplier/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+#. module: product_by_supplier
+#: view:product.supplierinfo:0
+msgid "Supplier Product Name"
+msgstr "Nom de l'article chez le fournisseur"
+
+#. module: product_by_supplier
+#: field:product.product,partner_ref2:0
+msgid "Supplier codes"
+msgstr "Codes Fournisseurs"
```
Comment (bwrsandman) [inline product_by_supplier/i18n/fr.po]: Codes Fournisseurs -> Codes fournisseurs

File: product_by_supplier/i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+#: view:product.supplierinfo:0
+msgid "Product Code"
+msgstr "Référence interne"
+
+#. module: product_by_supplier
+#: model:ir.actions.act_window,name:product_by_supplier.action_product_supplier_info
+#: model:ir.ui.menu,name:product_by_supplier.menu_product_supplier_info
+#: view:product.supplierinfo:0
+msgid "Product by Suppliers"
+msgstr "Articles par Fournisseurs"
```
Comment (bwrsandman) [inline product_by_supplier/i18n/fr.po]: Articles par Fournisseurs -> Articles par fournisseurs

File: product_by_supplier/i18n/fr.po [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+#. module: product_by_supplier
+#: field:product.supplierinfo,virtual_available:0
+msgid "Forecasted Quantity"
+msgstr "Quantité Prévue"
+
+#. module: product_by_supplier
+#: view:product.supplierinfo:0
+msgid "Group by..."
+msgstr "Grouper Par ..."
```
Comment (bwrsandman) [inline product_by_supplier/i18n/fr.po]: Grouper Par ... -> Grouper par ...

File: product_by_supplier/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: product_by_supplier
+#: field:product.supplierinfo,virtual_available:0
+msgid "Forecasted Quantity"
+msgstr "Quantité Prévue"
```
Comment (bwrsandman) [inline product_by_supplier/i18n/fr.po]: Quantité Prévue - > Quantité prévue

**Comment by bwrsandman** [comment]
In french, generally, important words aren't capitalized like they are in English.
I pointed out problems with capitalization in the translations.

**Comment by legalsylvain** [comment]
@bwrsandman Thanks.

**Comment by bwrsandman** [comment]
:+1:

**Comment by yvaucher** [comment]
error has nothing to do with this PR
:+1:

**Pull request merged**